catalogue/webapp/components/ItemRequestButton/ItemRequestButton.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import { type PhysicalItemAugmented } from '@weco/common/utils/works';
+// $FlowFixMe
```

Comment:
Nice way to do this! Could be worth leaving a `$FlowFixMe (tsx include)`?

Reply:
good shout